cl/users/tasks.py

```diff
@@ -34,78 +23,77 @@ def abort_or_retry(task, exc):
 @app.task(
     bind=True, max_retries=5, interval_start=5 * 60, interval_step=5 * 60
 )
-def subscribe_to_mailchimp(self, email):
-    path = f"/3.0/lists/{MC_LIST_ID}/members/"
+def subscribe_to_moosend(
+    self, email, mailing_list_id=settings.MOOSEND_DEFAULT_LIST_ID
```

Comment:
For now we don't need more than one email list, so I'd suggest this falls into YAGNI. Can you please remove this extra functionality?

Also, please add type hints.

Reply:
Ok, then I'll remove it and add the type hints. I didn't know "YAGNI" term until now, thanks for that.